css/style.css

```diff
@@ -0,0 +1,2912 @@
+  }
+
+  .charset__speed {
+    width: 65.92px;
+    height: 50.16px;
+    padding-left: 0;
+    margin-left: 20px;
+  }
+    
+  .middle__title {
+    margin-left: 230px;
+  }
+
+  .middle__info-title {
+    margin-left: 10px;
+  }
+
+  .middle__info-text {
+    margin-left: 10px;
+  }
+
+  .middle__first-title {
+    font-size: 17px;
+    padding-left: 10px;
+  }
+
+  .middle__first-text {
+    font-size: 10px;
+    padding-left: 10px;
+  }
+
+  .middle__btn {
+    margin-left: 30px;
+  }
+
+  .middle__text {
+   padding-left: 10px;
+   padding-right: 30px;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-size: 500px 100px;
+    background-position: 38px 68px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: 0 130px;
+  }
+
+}
+
+@media (max-width: 550px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .top {
+    padding: 90px 0px 30px;
+    padding-top: 26px;
+    background: url(../img/tesla-fon.png);
+    background-position: 0 160px;
+    background-repeat: no-repeat;
+    background-size: 550px 271px;
+  }
+
+  .top__wrapper-charset {
+    background: url(../img/svg/road-ipad.svg) no-repeat;
+    background-position: 270px 0;
+  }
+
+  .charset {
+    margin-right: 40px;
+    margin-left: 30px;
+  }
+
+  .charset__first {
+    width: 150px;
+  }
+
+  .charset__speed {
+    width: 65.92px;
+    height: 50.16px;
+    padding-left: 0;
+    margin-left: 0;
+  }
+  
+  .middle {
+    max-width: 550px;
+    margin-top: 0;
+  }
+
+  .middle__fon {
+    width: 100%;
+  }
+
+  .middle__title {
+    display: none;
+  }
+    
+  .middle__info {
+    max-width: 550px;
+    margin-top: 500px;
+    height: 400px;
+  }
+
+  .middle__info-row {
+    max-width: 550px;
+    height: 304px;
+  }
+
+  .middle__content {
+    max-width: 550px;
+  }
+
+  .middle__info-title {
+    margin-top: 18px;
+    margin-left: 24px;
+    margin-bottom: 10px;
+    font-size: 30px;
+    line-height: 37px;
+  }
+
+  .middle__info-text {
+    width: 300px;
+    margin-bottom: 0;
+    margin-top: 0px;
+    margin-left: 25px;
+    font-size: 15px;
+    line-height: 18px;
+  }
+
+  .middle__row {
+    display: none;
+  }
+
+  .button__btn {
+    display: none;
+  }
+
+  .bottom__content-stars {
+    width: 100%;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-size: 450px 100px;
+    background-position: 20px 100px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: 0 150px;
+  }
+
+}
+
+@media (max-width: 500px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .top {
+    padding: 90px 0px 30px;
+    padding-top: 26px;
+    background: url(../img/tesla-fon.png);
+    background-position: 0 160px;
+    background-repeat: no-repeat;
+    background-size: 500px 271px;
+  }
+
+  .top__wrapper { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)